docs/REMOTE_DEV.md

```diff
@@ -0,0 +1,67 @@
+# Happy Remote Development Environment
```

Comment:
can you also document somewhere how to use the new aspen cli commands that you wrote? (drop db, load data, setup remote db, etc)